src/xdocs/index.xml.vm

```diff
@@ -124,6 +124,51 @@
       </p>
     </section>
 
+    <section name="Limitations">
+
+          <p>
+              OK, so you have written your first Check, and you have found
+              several flaws in many of your programs. You now know that your
```

Comment:
such paragraph will looks weird on first page.

Lets have all limitation in original place and on first page just have a link to original place.
on first page we should have only one sentence.
`Checkstyle is single file static analysis tool, for more details please read <link to original place>full list</>`. 

@rnveach , please confirm.

Reply:
If you don't want the text on the first page, that is fine with me, but the text in the original place doesn't really make sense for a person who just wants to learn about our tool.

> OK, so you have written your first Check, and you have found


Someone coming from the front page won't have written a check and is just starting to learn about Checkstyle. If put a link on the front page, I think the text should be updated so it can be used by both people who are just learning about Checkstyle and those that are trying to dig deep into it.

Reply:
I am ok to remove the whole paragraph ` OK, so you have written ....` at original place.

@rnveach , are you ok with my proposal to have on index page one sentence only with link ?

Reply:
@romani Yes I am ok with it.

Reply:
@fzdy1914 , please help us to make it happen.

Reply:
I will work on it on weekends.